moulinette/authenticators/ldap.py

```diff
@@ -233,14 +233,33 @@ def validate_uniqueness(self, value_dict):
         Returns:
             Boolean | MoulinetteError
 
+        """
+        attr_found = self.get_conflict(value_dict)
+        if attr_found:
+            logger.info("attribute '%s' with value '%s' is not unique",
+                        attr_found[0], attr_found[1])
+            raise MoulinetteError(errno.EEXIST,
+                                  m18n.g('ldap_attribute_already_exists',
+                                         attribute=attr_found[0],
+                                         value=attr_found[1]))
+        return True
+
+    def get_conflict(self, value_dict, base_dn=None):
```

Comment:
What is the motivation in passing `base_dn` in args if it's not used yet ? Are there foreseen usage of this ?

Reply:
Because sometime I don't want all element in all part in the LDAP tree. By example want only to now in the group subtree  if there are a conflict.

Reply:
Okay I didn't realise this was a method for the auth object and that you are using this in https://github.com/YunoHost/yunohost/pull/585